docs/operate/get-started/basics.md

```diff
@@ -0,0 +1,99 @@
+
+`viam-server` connects your machine to the Viam app and to SDK clients.
+
+See [Viam architecture](/operate/reference/architecture/) for details on how Viam works.
+
+## What is a machine?
+
+A _machine_ is a computer (often a single-board computer like a Raspberry Pi or Jetson) or microcontroller and all the hardware attached to it, as well as the software running on it.
+You can think of one machine as representing one device, or one robot.
+Each machine runs an instance of `viam-server`.
+
+When you create a new machine in the Viam app, Viam generates a unique set of credentials for that machine that connect the physical machine to its instance in the Viam app.
+
+## What platforms does Viam run on?
+
+Viam can run on any computer that runs one of the following operating systems:
+
+- Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
+- macOS
+- Windows
+- 32-bit microcontrollers (limited support)
+
+Examples of computers that can run Viam:
+
+{{< board-carousel >}}
+
+## What hardware does Viam support?
+
+In short, anything.
+
+Viam supports a wide variety of sensors, cameras, and other physical hardware, with APIs for each of the following types of hardware:
+
+{{< cards >}}
+{{% relatedcard link="/dev/reference/apis/components/arm/" %}}
+{{% relatedcard link="/dev/reference/apis/components/base/" %}}
+{{% relatedcard link="/dev/reference/apis/components/board/" %}}
+{{% relatedcard link="/dev/reference/apis/components/button/" %}}
+{{% relatedcard link="/dev/reference/apis/components/camera/" %}}
+{{% relatedcard link="/dev/reference/apis/components/encoder/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gantry/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gripper/" %}}
+{{% relatedcard link="/dev/reference/apis/components/input-controller/" %}}
+{{% relatedcard link="/dev/reference/apis/components/motor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/movement-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/power-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/servo/" %}}
+{{% relatedcard link="/dev/reference/apis/components/switch/" %}}
+{{% relatedcard link="/dev/reference/apis/components/generic/" %}}
+{{< /cards >}}
+
+These standardized APIs are implemented by {{< glossary_tooltip term_id="module" text="modules" >}} that provide drivers for specific models of hardware.
+
+Any hardware that is not already supported by a Viam module can be added into Viam's system of modular resources by [creating a new module](/operate/get-started/other-hardware/) that provides a driver for the hardware.
+
+Viam also supports various software services such as [data capture](/data-ai/capture-data/capture-sync/) and [computer vision](/dev/reference/apis/services/vision/), designed to integrate seamlessly with the hardware driver modules.
```

Comment:
```suggestion
```

Suggested moving examples of software services up into the bullets, both to surface them earlier, and because this section is otherwise about hardware.

I also think data capture may eventually become a "system service" rather than an API - it doesn't really make sense to have multiple implementations of it. I suppose people could implement data capture to their own cloud, but the API is pretty specific to the way Viam's cloud is designed.

I think computer vision is a good example, we could either additionally include motion planning or leave it out if that's too much emphasis on motion planning.